src/integrationtest/java/com/ericsson/ei/frontend/CommonSteps.java

```diff
@@ -139,6 +140,15 @@ public void request_sent() throws Throwable {
         response = httpRequest.performRequest();
     }
 
+    @When("^request is sent for (\\d+) seconds until reponse code no longer matches (\\d+)$")
+    public void request_sent_body_not_received(int seconds, int statusCode) throws Throwable {
+        long stopTime = System.currentTimeMillis() + seconds * 1000;
```

Comment:
long stopTime = System.currentTimeMillis() + seconds * 1000;
is exactly the same as 
long stopTime = System.currentTimeMillis() + (seconds * 1000);

Although for someone just reading the function quickly the second line is more clear to how the calculation is done.

Reply:
Done